src/app.module.ts

```
@@ -64,6 +66,20 @@ dotenv.config();
+      throttlers: [
+        {
+          ttl: 60 * 1000, // 1 minute in ms
+          limit: 100,
+        },
+      ],
+    }),
+  ],
+  providers: [
+    {
+      provide: APP_GUARD,
+      useClass: require('./auth/guards/rate-limit.guard').RateLimitGuard,
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)